### DISCUSSION TITLE
Deploy with Node.js for v4

### DISCUSSION THREAD

**Discussion by pietryszak** [lowdefy/lowdefy#1665] [Docs]
During lack of instruction on how to deploy 4.0.0.-rc.10 any suggestion how to do that step by step will be helpfull :)

**Comment by Gervwyk** [comment]
Hi @pietryszak, Lowdefy v4 is just a Nextjs app, so any deployment solution that works for Nextjs should be doable.

With that said, I see that the lowdefy docs for nodejs deployment is just a todo, sorry, we'll fix that.

In the mean time I can refer you to the Nextjs docs for deploying using a node server. https://nextjs.org/docs/pages/building-your-application/deploying#nodejs-server

You can use the lowdefy build cli command to create the production build output, which is the nextjs app to deploy to your production environment.

Does this help?
